package.json

```diff
@@ -51,11 +51,12 @@
     "reset:env": "./script/reset-environment.sh",
     "security-check": "node ./script/security-check.js",
     "test": "npm run test:unit",
-    "test:contract": "BUILDTYPE=localhost npm run test:unit -- --reporter mochawesome --config config/mocha-pact.json 'src/**/tests/**/*.pact.spec.js'",
+    "test:contract": "npm run test:unit-pact-config 'src/**/tests/**/*.pact.spec.js'",
```

Comment:
I kept this so that there's still an easy way to run all contract tests if needed, even though this won't be used in the CI workflow